workflow/rules/read_clipping.smk

```diff
@@ -10,7 +10,6 @@ rule clip_primer:
         ),
     output:
         sortbam=temp("results/{date}/clipped-reads/{sample}.bam"),
-        sortindex=temp("results/{date}/clipped-reads/{sample}.bam.bai"),
```

Comment:
Dont we need the index here?

Reply:
This was somehow double posted. This closing this.

Reply:
Do or don't?